builtins/nlohmann/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 # file only used when ACLiC or ROOT macros will include REve headers,
 # it is not used for ROOT compilation
 
+set(nlohmann_json_VERSION "3.9.1" PARENT_SCOPE)
```

Comment:
Let's change here from the hardcoded version? You can just use  directly idea from LZ4 https://github.com/root-project/root/blob/master/builtins/lz4/CMakeLists.txt#L31-L39

Reply:
I add commit, but it increase build time. 
And this version is irrelevant - one not need to search nlohmann_json when using buildin version. 
I put it here just to avoid empty argument in ROOTConfig.cmake file

Reply:
It is nice not to have a hardcoded version,  so next time when somebody will upgrade nlohmann_json and forgot to fix it will be safe.

Reply:
I will keep it if you like it - it does not play any role.